modules/mono/glue/GodotSharp/GodotSharp/Core/Aabb.cs

```diff
@@ -509,6 +509,62 @@ public readonly bool IntersectsPlane(Plane plane)
             return under && over;
         }
 
+        /// <summary>
+        /// Returns <see langword="true"/> if the <see cref="Aabb"/> intersects
+        /// the ray along <paramref name="dir"/> positioned at <paramref name="from"/>.
+        /// </summary>
+        /// <param name="origin">The origin of the ray.</param>
+        /// <param name="dir">The direction of the ray.</param>
+        /// <returns>
+        /// A <see langword="bool"/> for whether or not the <see cref="Aabb"/> intersects the ray.
+        /// </returns>
+        public readonly bool IntersectsRay(Vector3 from, Vector3 dir)
+        {
+            if (HasPoint(from)) return true;
+
+            real_t tmin = -1e20f;
+            real_t tmax = 1e20f;
+
+            for (int i = 0; i < 3; i++)
+            {
+                if (dir[i] == 0)
+                {
+                    if ((from[i] < Position[i]) || (from[i] > End[i]))
```

Comment:
We usually use the `_position` field instead of the property. This is just a matter of code-style to be consistent with the rest of the file, because the JIT will optimize this anyway.

```suggestion
                    if ((from[i] < _position[i]) || (from[i] > End[i]))
```